docs/api_v2/api_v2_reference.apib

```diff
@@ -1938,6 +1938,144 @@ In order to save new items to the dataset, send HTTP POST request with JSON payl
+
++ Parameters
+
+    + runId: `3KH8gEpp4d8uQSe8T` (string, required) - Run ID.
+    + token: `rWLaYmvZeK55uatRrZib4xbZs` (string, required) - API authentication token.
+
++ Response 200 (application/json)
+
+    + Attributes
+        - data (RunAborted, required)
+
+## Metamorph run [/v2/actor-runs/{runId}/metamorph{?token,build,targetActorId}]
+
+### Metamorph run [POST]
+
+Transforms an actor run into a run of another actor with a new input.
+This is useful if you want to use another actor to finish the work
+of your current actor run, without the need to create a completely new run and waiting for its finish.
+For the users of your actors, the metamorph operation is transparent, they will just see your actor got the work done.
+
+Internally, the system stops the Docker container corresponding to the actor run
+and starts a new container using a different Docker image.
+All the default storages are preserved and the new input is stored under the `INPUT-METAMORPH-1` key in the same default key-value store.
+
+For more information, see [Actor docs](https://docs.apify.com/actors/development/source-code#metamorph).
+
++ Parameters
+
+    + runId: `3KH8gEpp4d8uQSe8T` (string, required) - Actor run ID.
+    + token: `rWLaYmvZeK55uatRrZib4xbZs` (string, required) - API authentication token.
+    + targetActorId: `HDSasDasz78YcAPEB` (string, required) - ID of a target actor that the run should be transformed into.
+    + build: `beta` (string, optional) - Optional build of the target actor.
+      It can be either a build tag or build number. By default, the run uses the build specified in the default run configuration for the target actor (typically `latest`).
+
++ Response 200 (application/json)
+
+    + Attributes
+        - data (RunMetamorphed, required)
+
+## Resurrect run [/v2/actor-runs/{runId}/resurrect{?token}]
+
+### Resurrect run [POST]
+
+Resurrects a finished actor run and returns an object that contains all the details about the resurrected run.
+Only finished runs, i.e. runs with status `FINISHED`, `FAILED`, `ABORTED` and `TIMED-OUT` can be resurrected.
+Run status will be updated to RUNNING and its container will be restarted with the same storages
+(the same behaviour as when the run gets migrated to the new server).
+
+For more information, see [Actor docs](https://docs.apify.com/actor/run#resurrection-of-finished-run).
+
++ Parameters
+
+    + runId: `3KH8gEpp4d8uQSe8T` (string, required) - Run ID.
+    + token: `rWLaYmvZeK55uatRrZib4xbZs` (string, required) - API authentication token.
+
++ Response 200 (application/json)
+
+    + Attributes
+        - data (Run, required)
+
+# Group Actor builds
+
+The API endpoints described in this section enable you to manage Apify actor build.
+
+## Build object [/v2/actor-builds/{buildId}{?token,waitForFinish}]
+
+### Get build [GET]
+
+Gets an object that contains all the details about a specific build of an actor.
```

Comment:
Same as above: would it not be good to also move Build Collection to here?
https://docs.apify.com/api/v2#/reference/actors/build-collection

Reply:
We plan to move list od builds there as well, but for now, we moved just build object as it a prerequisite for javascript client v1.